docs/document/website/versioned_docs/version-1.4.0/getting-started/quick-start.md

```diff
@@ -0,0 +1,195 @@
+
+### Example for Desktop components
+
+![desktop components](assets/desktop_components.png)
+
+### Example for Mobile components
+
+![mobile components](assets/mobile_components.png)
+
+## Preparation: Create App
+
+In this tutorial, you will place a Button component inside the header menu space on the Record List page of a Kintone app.
+
+First, follow the below steps to create an app to install Kintone UI Component.
+
+1. Create a blank app. ([Creating an App from Scratch](https://get.kintone.help/k/en/user/create_app/tutorial.html))<br>
+You do not need to set up any fields in this app.
+
+## Installation
+
+Kintone UI Component supports installation via a UMD, a CDN, or a npm package.<br>
+You can choose to import or implement the library according to the environment such as the browser environment or Node.js environment.<br>
+In this article, we will show you how to install and implement using each approach.
+
+> See the Components section in the sidebar for more details on each component.<br>
+> (Additional components will be added in order.)
+
+### Use the UMD
+> We recommend that you should upgrade Kintone UI Component to the latest version for your projects. <br>
+> From v1.4.0, please use `Kucs["1.x.x"]` instead of `Kuc` and specify your expected version (ex. `Kucs["1.4.0"]`).<br>
+>The rendered components' tags and class names will include the version number.<br>
+> When using a version before v1.4.0, please use `Kuc` as a global variable but **note that it may be conflicting when adding two or more `kuc.min.js` files** on Kintone app plug-in or customization.
+> Please read this [Version Conflicts Issue and Solution](../guides/version-conflicts-issue-solution) article for more information!
+
+1. Download the file located inside Kintone UI Component repository's [each version Release field](https://github.com/kintone-labs/kintone-ui-component/releases). Unzip the attached archives folder (kintone-ui-component-{version} .tgz). Upload the following file to the `JavaScript and CSS Customization` inside Kintone's app setting.
+
+```text
+./umd/kuc.min.js
+```
+
+2. Create a `index.js` file.
+
+```js
+const Kuc = Kucs["1.x.x"];
+
+kintone.events.on('app.record.index.show', event => {
+  const header = kintone.app.getHeaderMenuSpaceElement();
+
+  const button = new Kuc.Button({
+    text: 'Submit',
+    type: 'submit'
+  });
+  button.addEventListener('click', event => {
+    console.log(event);
+  });
+
+  header.appendChild(button);
+  return event;
+});
+```
+
+3. To apply the customization, upload `index.js` file to the `JavaScript and CSS Customization` inside the Kintone app settings. ([Customizing an App with JavaScript and CSS](https://get.kintone.help/k/en/user/app_settings/js_customize.html))
+
+![button customize](assets/button_customize.png)
+
+### Use the CDN
+> Following the [Version Conflicts Issue and Solution](../guides/version-conflicts-issue-solution#using-cdn) guide, please use `Kucs['1.x.x']` instead of `Kuc`.
```

Comment:
```suggestion
> Following the [Version conflicts issue and solution](../guides/version-conflicts-issue-solution#using-cdn) guide, please use `Kucs['1.x.x']` instead of `Kuc`.<br>
```